tasks/copy.js

```diff
@@ -19,6 +19,13 @@ module.exports = {
     return src("src/**/**/*.scss").pipe(dest("dist/scss"));
   },
 
+  // Copy material icons to /dist/img/material-icons
+  copyIcons() {
+    dutil.logMessage("copyIcons", "Copying Material icons to dist/img/material-icons");
+    return src(["node_modules/@material-design-icons/svg/outlined/*"])
```

Comment:
Do you imagine this being run often? It seems like a lot to copy continually. We could look into gulp's [`lastRun()`](https://gulpjs.com/docs/en/api/lastrun) function to speed up build times.

If the files didn't change, it won't run again.

```js
copyIcons() {
    dutil.logMessage("copyIcons", "Copying Material icons to dist/img/material-icons");
    return src(["node_modules/@material-design-icons/svg/outlined/*"], since: lastRun(copyIcons))
      .pipe(dest("dist/img/material-icons"));
  }
```

---

Might be worth trying, if we haven't already, to see if we can add `@material-design-icons`` as a `const` above and try to shorten the path like `material-icons/outlined/*`